src/Be.Vlaanderen.Basisregisters.Projector/ProjectionsManagerHelpers.cs

```diff
@@ -33,5 +34,20 @@ public static class ProjectionsManagerExtensions
 
             return app;
         }
+
+        public static IApplicationBuilder UseProjectionsManagerAsync(
+            this IApplicationBuilder app,
+            ProjectionsManagerOptions options)
+        {
+            options.Common.ApplicationLifetime.ApplicationStopping.Register(() => ProjectionsCancellationTokenSource.Cancel());
+
+            Task.Run(() =>
```

Comment:
Task.Run returns a Task, which must be either `await`ed or `.Wait()`ed on

Reply:
but if we wait, we introduce the same problem

Reply:
added cancellation token to the `Task`, so when the host stops. This also is cancelled.